From 7ff54ae9b68ca0edf3d6cd57c3eefb6cf072b6f8 Mon Sep 17 00:00:00 2001
From: Ahmad Samir <a.samirh78@gmail.com>
Date: Fri, 19 Aug 2022 22:50:26 +0200
Subject: [PATCH] Disable deprecated KF API from the latest KF released version

This unbreaks building while porting to new API happens.

GIT_SILENT

(cherry picked from commit caa66e6a930a6b892987961a42ea35f5b9b1aac4)
---
 CMakeLists.txt | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5c62156..9af9cef 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -40,8 +40,9 @@ ecm_setup_version(PROJECT
                   PACKAGE_VERSION_FILE "${CMAKE_CURRENT_BINARY_DIR}/KF5BalooWidgetsConfigVersion.cmake")
 
 add_definitions(-DQT_DISABLE_DEPRECATED_BEFORE=0x050F00)
-add_definitions(-DKF_DISABLE_DEPRECATED_BEFORE_AND_AT=0x060000)
-
+# Bump manually after building with the latest KF released version
+# and porting all deprecated code
+add_definitions(-DKF_DISABLE_DEPRECATED_BEFORE_AND_AT=0x056100) # 5.97.0
 add_subdirectory(src)
 if (BUILD_TESTING)
   add_subdirectory(test)
-- 
GitLab

